rpc/rpc.go

```diff
@@ -60,6 +60,45 @@ func (s *server) GetSlotAndShardAssignment(ctx context.Context, in *pb.GetSlotAn
 	return &pb.SlotAndShardAssignment{ShardID: shardID, Slot: slot, Role: r}, nil
 }
 
+func validatorToPb(validator *primitives.Validator) *pb.ValidatorResponse {
```

Comment:
Move this to the primitives package and make a method of `Validator` struct `ToProto`.